src/tests/test_common/homestore_test_common.hpp

```diff
@@ -190,14 +192,35 @@ class HSTestHelper {
         }
 
         std::vector< homestore::dev_info > device_info;
-        if (SISL_OPTIONS.count("device_list")) {
-            s_dev_names = SISL_OPTIONS["device_list"].as< std::vector< std::string > >();
+        if (!cust_dev_names.empty() || SISL_OPTIONS.count("device_list")) {
+            cust_dev_names.empty() ? s_dev_names = SISL_OPTIONS["device_list"].as< std::vector< std::string > >()
+                                   : s_dev_names = std::move(cust_dev_names);
             LOGINFO("Taking input dev_list: {}",
                     std::accumulate(
                         s_dev_names.begin(), s_dev_names.end(), std::string(""),
                         [](const std::string& ss, const std::string& s) { return ss.empty() ? s : ss + "," + s; }));
 
+            // zero the homestore pdev's first block for each device;
+            auto const zero_size = hs_super_blk::first_block_size();
```

Comment:
suggesting move the initialization out of the UT.
Or at least we have a flag whether to zero out,  otherwise `restart_homestore` will not work